Importers/Rhino/Speckle.Importers.Rhino/Internal/IPCModels.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Speckle.Sdk.Credentials;
+using Version = Speckle.Sdk.Api.GraphQL.Models.Version;
+
+namespace Speckle.Importers.Rhino.Internal;
+
+//Needs to be aligned between JobProcess and Rhino Importer
+internal readonly struct ImporterArgs
+{
+  public string FilePath { get; init; }
+  public string ResultsPath { get; init; }
+  public string ProjectId { get; init; }
+  public string ModelId { get; init; }
+  public Account Account { get; init; }
+}
+
+public readonly struct ImporterResponse
+{
+  public Version? Version { get; init; }
+  public string? ErrorMessage { get; init; }
+}
```

Comment:
These two structs are copy pasted between the two projects.
A little nasty, but the alternatives are also not too pretty
- 🤪 These live somewhere else (like the SDK or Speckle.Connector.Common, or a new project... pretty nasty) 
- 🤮 We use dynamic definitions (infinitely worse than this)
- 🤷 We have a direct reference JobProcessor -> RhinoImporter... perhaps not too nasty, but I wanted to avoid coupling them directly.

Happy to consider the latter if these two structs offend ppl living in two places.